tools/template/README.md

```diff
@@ -0,0 +1,43 @@
+# wtstats template
+
+This subfolder contains the code to generate the `wtstats.html.template` file needed by the Python script `wtstats.py`, which is located in the `./tools` directory. 
+
+The template is generated as a node.js "single page" application using several open source frameworks, see Dependencies below. The `wtstats.py` Python script parses stats files, transforms the data and inserts it as JSON into the template. The resulting HTML page can be openend in the browser.
+
+### Build Process
+
+To build the template, follow these steps: 
+
+1. change into the `./tools/template` directory
+2. run `npm install` to install all missing dependencies. 
```

Comment:
It also isn't on AWS Linux by default nor does yum know where to find it:
```
% sudo yum install npm
...
No package npm available.
Error: Nothing to do
```

The history of wtstats.py went from having to find/download and install nvd3, which was painful, to Michael/Alex creating a self-contained set of tools including nvd3 existing in `tools/3rdparty` and it all worked out of the box.  Now we're back to having to find/download both the package manager and the things it needs.

I assume what npm installs are all the things listed in `package.json`.  Is `package.json` generated?  How difficult is it going to be to maintain and keep up with all those specific versions of each of those items (maybe 20-30 of them)?